pkgs/by-name/re/recaf/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  maven,
+  fetchFromGitHub,
+  jdk11,
+  makeBinaryWrapper,
+  libGL,
+  glib,
+  gtk3,
+  libXtst,
+  nix-update-script,
+}:
+let
+  version = "2.21.14";
+  jdk = jdk11;
+in
+maven.buildMavenPackage {
+  pname = "recaf";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "Col-E";
+    repo = "Recaf";
+    rev = "refs/tags/${version}";
+    hash = "sha256-aVOVfrACawYc9VlBZ6+IHiCDGPOsVLdAvwQHdByZ584=";
+  };
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  mvnHash = "sha256-MPPZXQilmMaj8n5SXP7fG1Q95pH5Ydu7jU+KRS9beO8=";
```

Comment:
For some reason, this hash is different on ARM.
`sha256-r7yWnPAuk4zp11eP8D3QG2zP5qa/2lPsfWxOLuGbC8U=` vs. the value above.

Reply:
Yup, we may be hitting some limits of the Maven feature. Would be good to track this down, otherwise mark the package as broken on ARM

(macOS also fails at building this period)